docs/asciidoc/modules/ROOT/partials/usage/apoc.load.directory.async.adoc

```diff
@@ -1,31 +1,8 @@
-We can pass from 2 to 5 parameter in the `apoc.load.directory.async.add`.
+=== Adding a directory listener
 
-The first parameter is the name of our custom watch listener.
-If we use an already existing listener name, that listener will be overwritten.
-The second parameter is the cypher query that will be executed.
-The query can have the following parameters:
+With other load procedures such as `apoc.load.csv` or `apoc.load.json`, it is possible to load files asynchronously.
 
-* $fileName: the name of the file which triggered the event
-* $filePath: the absolute path of the file which triggered the event if `apoc.import.file.use_neo4j_config=false`, otherwise the relative path starting from `$IMPORT_DIR`
-* $fileDirectory: the absolute path directory of the file which triggered the event if `apoc.import.file.use_neo4j_config=false`, otherwise the relative path starting from `$IMPORT_DIR`
-* $listenEventType: the triggered event ("CREATE", "DELETE" or "MODIFY"). The event "CREATE" happens when a file is inserted in the folder,
-"DELETE" when a file is removed from the folder and "MODIFY" when a file in the folder is changed.
-Please note that if a file is renamed, will be triggered 2 event, that is first "DELETE" and then"CREATE"
-
-The third parameter is the pattern of file to search for.
-By default is '*', that is, search all files and it leverages the https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/filefilter/WildcardFileFilter.html[WildcardFileFilter - Apache Commons IO].
-
-The fourth is the search path of directory.
-By default is an empty string, that is, search file in `import` directory.
-
-
-=== Examples of interoperability with other procedures
-
-We can use another load procedure as query parameter, like `apoc.load.csv` or `apoc.load.json`.
-
-Here are two examples.
-
-We can execute:
```

Comment:
Again depending on if you can actually change the auto-generated files or not, you might still want to capture this info somewhere in this file